spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/TaskDefinitionRepository.java

```diff
@@ -32,7 +32,7 @@ public interface TaskDefinitionRepository extends KeyValueRepository<TaskDefinit
 
 	Page<TaskDefinition> findByTaskNameContains(String taskName, Pageable pageable);
 
-	Page<TaskDefinition> findByTaskNameContainsAndDslTextContains(String taskName, String dslText, Pageable pageable);
```

Comment:
Unfortunately I think we should this other (seemingly unused) method and just mark it as `@deprecated`. It is not directly related to this PR and leaving it would reduce any risk for this proposal.